test/e2e/specs/editor/various/draggable-blocks.spec.js

```diff
@@ -4,12 +4,8 @@
 const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
 
 test.describe( 'Draggable block', () => {
-	test.beforeEach( async ( { admin, page } ) => {
+	test.beforeEach( async ( { admin } ) => {
 		await admin.createNewPost();
-		await page.setViewportSize( {
```

Comment:
@kevin940726, is the default viewport height enough to prevent scrolling? The original test includes a comment about this.

Reply:
TBH, I'm not entirely sure. I _think_ Playwright will make sure scrolling doesn't occur when starting the drag by its actionability check. I'd say let's remove this unless we know this still happens in Playwright.

Reply:
Added this back again.